Sources/Common/Service/CIOApiEndpoint.swift

```diff
@@ -33,19 +35,23 @@ public enum CIOApiEndpoint {
         case .trackCustomerEvent: return "POST"
         case .pushMetrics: return "POST"
         case .trackDeliveryMetrics: return "POST"
+        case .trackNSEPushDeliveryMetrics: return "POST"
         }
     }
 }
 
 public extension CIOApiEndpoint {
-    func getUrl(baseUrls: HttpBaseUrls) -> URL? {
-        URL(string: getUrlString(baseUrls: baseUrls))
+    func getUrl(baseUrl: String) -> URL? {
+        URL(string: getUrlString(baseUrl: baseUrl))
     }
 
-    func getUrlString(baseUrls: HttpBaseUrls) -> String {
+    func getUrl(baseUrls: HttpBaseUrls) -> URL? {
```

Comment:
this is going to go away in later PRs, as we don't use it anymore. keeping in this PR so the build remains compilable if anyone wants to run it.